integration-tests/src/test/java/com/datastax/oss/driver/core/ssl/ProgrammaticSslIT.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
*NIT* It seems like we are inconsistent with our naming, other classnames in our project use SSL,  instead of Ssl we should try to bring the naming inline so we are consistent.

Reply:
Apart from `SSLContext` (which is not ours, it comes from the JDK), I think we used `Ssl` everywhere.